client/packages/common/src/intl/locales/pt/replenishment.json

```diff
@@ -31,8 +31,8 @@
   "label.hide-stock-over-minimum": "Ocultar estoque acima do mínimo",
   "label.max": "Máximo",
   "label.min": "Mínimo",
-  "label.max-months-of-stock": "Máximo MDE (Meses De Estoque)",
```

Comment:
These are again best guesses at translations. (not reviewed by a native speaker)